microsoft/testsuites/dpdk/dpdkperf.py

```diff
@@ -76,41 +76,43 @@ def perf_dpdk_failsafe_pmd_multi_core(
         variables: Dict[str, Any],
     ) -> None:
 
-        self._run_dpdk_perf_test("failsafe", result, log, variables, use_max_cores=True)
+        self._run_dpdk_perf_test("failsafe", result, log, variables, service_cores=4)
 
     @TestCaseMetadata(
         description="""
-        DPDK Performance: failsafe mode, maximal core count, default queue settings
+        DPDK Performance: failsafe mode, multiple service cores, max nics
         """,
         priority=3,
         requirement=simple_requirement(
             min_count=2,
             network_interface=Sriov(),
-            min_nic_count=2,
-            min_core_count=48,
+            min_nic_count=8,
+            min_core_count=64,
```

Comment:
I'm almost done the IsolatedResource feature (called exclusive VM). Do you mind change all "min_core_count=64" to this feature? The source is from https://docs.microsoft.com/en-us/azure/security/fundamentals/isolation-choices#compute-isolation. And if new VM sizes are not here, it can be added separatedly.

Reply:
Ah I see this was done for me.